genie-agent/src/test/groovy/com/netflix/genie/agent/execution/exceptions/LockExceptionSpec.groovy

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Take a look at AgentExecutionExceptionSpec.groovy

Exception tests are dull so there's one test class to test all of them, just add one line to that file and you can get rid of this one

Reply:
Done